### ISSUE TITLE
Version 1.0.2 does not have Badge

### ISSUE THREAD

**Issue by long-nguyentien** [thrive-now/BottomNavigationBarXF#67]
Thank you for your great packet. However please add newest version to Nuget which contains Badge because current version does not have that feature.

**Comment by c0rnh0li0** [comment]
Yes, can confirm BottomBarPageExtensions has only two methods available now, GetTabColor and SetTabColor. There is no way adding a badge at this point. Thank you.

**Comment by XXISerenaIXX** [comment]
Yes, it it the same here. Can't use any of SetBadgeCount, GetBadgeCount, SetBadgeColor, GetBadgeColor .

**Comment by OpticNectar** [comment]
Anyone find a way to get Badges working again?

**Comment by mominali12** [comment]
So what's the solution?

**Comment by madkarlsson** [comment]
Since they haven't released the new version of the nuget the way to fix it is to fork/download and build the latest yourself. Do a release build and then include the DLL in your project (and the android DLL in any android projects).

Not ideal and very poor support from thrive regarding this sadly
